### PR TITLE
2.13.0: Changelog, Upgrading, VERSION

### PR DESCRIPTION
**Pull request by N-o-X**
This PR adds the 2.13.0 Changelog, upgrading documentation and increases the version in our VERSION file.

### CODE REVIEW

File: doc/16-upgrading-icinga-2.md [hunk 1]
```diff
@@ -8,6 +8,27 @@ Specific version upgrades are described below. Please note that version
...
+### DB IDO Schema Update <a id="upgrading-to-2-13-db-ido"></a>
+
+There is an optional schema update on MySQL which increases the max length of object names from 128 to 255 characters.
+
+Please proceed here for the [MySQL upgrading docs](16-upgrading-icinga-2.md#upgrading-mysql-db).
+
+### Behavior changes <a id="upgrading-to-2-13-behavior-changes"></a>
+
+#### Deletion of child downtimes on services
+Service downtimes created while using the `all_services` flag on the [schedule-downtime](https://icinga.com/docs/icinga-2/snapshot/doc/12-icinga2-api/#schedule-downtime) API action will now automatically be deleted when deleting the hosts downtime.
```
Comment (Al2Klimov) [inline doc/16-upgrading-icinga-2.md:22]: ... but not here?

File: doc/16-upgrading-icinga-2.md [hunk 2]
```diff
@@ -8,6 +8,27 @@ Specific version upgrades are described below. Please note that version
...
+
+### Behavior changes <a id="upgrading-to-2-13-behavior-changes"></a>
+
+#### Deletion of child downtimes on services
+Service downtimes created while using the `all_services` flag on the [schedule-downtime](https://icinga.com/docs/icinga-2/snapshot/doc/12-icinga2-api/#schedule-downtime) API action will now automatically be deleted when deleting the hosts downtime. 
+
+#### Windows Event Log
+Icinga 2.13 now supports logging to the Windows Event Log. Icinga will now also log messages from the early startup phase to the Windows Event Log. These were previously missing from the log file and you could only see them by manually starting Icinga in the foreground.
+
+This feature is now enabled and replaces the existing mainlog feature logging to a file. When upgrading, the installer will enable the windowseventlog feature and disable the mainlog feature. Logging to a file is still possible. If you don't want this configuration migration on upgrade, you can opt-out by installing the %ProgramData%\icinga2\etc\icinga2\features-available\windowseventlog.conf file before upgrading to Icinga 2.13.
```
Comment (Al2Klimov) [inline doc/16-upgrading-icinga-2.md:27]: I'd like to have that path inside \` \`.

File: doc/16-upgrading-icinga-2.md [hunk 3]
```diff
@@ -8,6 +8,27 @@ Specific version upgrades are described below. Please note that version
...
+
+#### Broken API package name validation
+This version has replaced a broken regex in the API package validation code which results in package names now being validated correctly. Package names should now only consist of alphanumeric characters, dashes and underscores. This change only applies to newly created packages to support already existing ones.
```
Comment (Al2Klimov) [inline doc/16-upgrading-icinga-2.md:30]: I'd prefer long lines to be separated with a soft limit of 80, keeping the text easily readable (i.e. split before or after, but not between "package names").

**Review by Al2Klimov — CHANGES_REQUESTED** [review]

File: doc/16-upgrading-icinga-2.md (hunk 1 above)
Comment (Al2Klimov) [reply doc/16-upgrading-icinga-2.md]: And here?

https://github.com/Icinga/icinga2/pull/8944/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR14-R15

Comment (N-o-X) [reply doc/16-upgrading-icinga-2.md]: Fixed

**Review by Al2Klimov — APPROVED** [review]

**Pull request merged**